packages/plugins/src/components/plugin-area/index.js

```diff
@@ -59,12 +59,13 @@ class PluginArea extends Component {
 
 	getCurrentPluginsState() {
 		return {
-			plugins: map( getPlugins(), ( { icon, name, render } ) => {
+			plugins: map( sortBy( getPlugins(), [ 'priority' ] ), ( { icon, name, render, priority } ) => {
```

Comment:
Maybe we might compose these 2 with lodash `flow`?

Reply:
I've changed this to use `compose`.

Reply:
great, it reads easier now, thanks